auth/feature-tests/vitest-step-definitions/GovUKMobilePostAuthenticationFunction.steps.ts

```diff
@@ -0,0 +1,49 @@
+import { loadFeature, describeFeature } from "@amiceli/vitest-cucumber";
```

Comment:
We will probably want to wrap this into a generic test for all lambdas to ensure we're being consistent